plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/packagematerial/JsonMessageHandler1_0.java

```diff
@@ -159,7 +159,10 @@ public String requestMessageForLatestRevision(PackageConfiguration packageConfig
 
     @Override
     public PackageRevision responseMessageForLatestRevision(String responseBody) {
-        return toPackageRevision(responseBody);
+        PackageRevision packageRevision = toPackageRevision(responseBody);
+        if( packageRevision == null) {
+            throw new RuntimeException("Empty response body");
+        } else return packageRevision;
```

Comment:
Missing `{}` block — we prefer to not if-else without blocks. See http://softwareengineering.stackexchange.com/a/16530